File: Estruturas_pre-definidas/supermercados-cpp/usuario.cpp
#include <iostream>
#include <string>
#include <vector>
#include "usuario.h"
using namespace std;


Usuario::Usuario(){}

Usuario::Usuario(string nome, string cpf){
    this->nome = nome;
    this->cpf = cpf;
}

string Usuario::getCpf(){
    return this->cpf;
}

string Usuario::getNomeCliente(){
    return this->nome;
}

File: Estruturas_pre-definidas/supermercados-java/Usuario.java
import java.util.Scanner;
import java.util.ArrayList;

public class Usuario {
    private String nome;
    private String cpf;

    public Usuario(){}

    public Usuario(String nome, String cpf){
        this.nome = nome;
        this.cpf = cpf;
    }

    public String getCpf(){
        return this.cpf;
    }

    public String getNomeCliente(){
        return this.nome;
    }
}